share/revng/test/tests/analysis/DetectABI/x86_64/propagate.S.model.yml

```diff
@@ -0,0 +1,24 @@
+#
+# This file is distributed under the MIT LIcense. See LICENSE.md for details.
+#
+
+---
+Functions:
+  - Entry: "0x400660:Code_x86_64"
```

Comment:
Can't hardcode addresses in ground truth.

Can you put `CustomName`?

Reply:
This doesn't contain `CustomName` in model received from `DetectABI`.